packages/core/hooks/useFileAccessContextMenu.ts

```diff
@@ -1,12 +1,22 @@
 import { ContextualMenuItemType, IContextualMenuItem } from "@fluentui/react";
 import * as React from "react";
 import { useDispatch, useSelector } from "react-redux";
+import { shell } from "electron";
```

Comment:
`electron` is not available on web so this would break our web deploy. Platform specific dependencies have to go into the platform specific package so `electron` stuff would go into `desktop`

Reply:
Good point! Addressed in 0b8c222201b63d8dd6c926f1a52f6e51f39cd10d , moved files around in 0b8c222201b63d8dd6c926f1a52f6e51f39cd10d